cmd/jujud/agent/machine/manifolds.go

```diff
@@ -619,24 +683,28 @@ const (
 	migrationInactiveFlagName = "migration-inactive-flag"
 	migrationMinionName       = "migration-minion"
 
-	servingInfoSetterName         = "serving-info-setter"
-	apiWorkersName                = "unconverted-api-workers"
-	rebootName                    = "reboot-executor"
-	loggingConfigUpdaterName      = "logging-config-updater"
-	diskManagerName               = "disk-manager"
-	proxyConfigUpdater            = "proxy-config-updater"
-	apiAddressUpdaterName         = "api-address-updater"
-	machinerName                  = "machiner"
-	logSenderName                 = "log-sender"
-	deployerName                  = "unit-agent-deployer"
-	authenticationWorkerName      = "ssh-authkeys-updater"
-	storageProvisionerName        = "storage-provisioner"
-	resumerName                   = "mgo-txn-resumer"
-	identityFileWriterName        = "ssh-identity-writer"
-	toolsVersionCheckerName       = "tools-version-checker"
-	machineActionName             = "machine-action-runner"
-	hostKeyReporterName           = "host-key-reporter"
-	fanConfigurerName             = "fan-configurer"
-	externalControllerUpdaterName = "external-controller-updater"
-	globalClockUpdaterName        = "global-clock-updater"
+	servingInfoSetterName           = "serving-info-setter"
+	apiWorkersName                  = "unconverted-api-workers"
+	rebootName                      = "reboot-executor"
+	loggingConfigUpdaterName        = "logging-config-updater"
+	diskManagerName                 = "disk-manager"
+	proxyConfigUpdater              = "proxy-config-updater"
+	apiAddressUpdaterName           = "api-address-updater"
+	machinerName                    = "machiner"
+	logSenderName                   = "log-sender"
+	deployerName                    = "unit-agent-deployer"
+	authenticationWorkerName        = "ssh-authkeys-updater"
+	storageProvisionerName          = "storage-provisioner"
+	resumerName                     = "mgo-txn-resumer"
+	identityFileWriterName          = "ssh-identity-writer"
+	toolsVersionCheckerName         = "tools-version-checker"
+	machineActionName               = "machine-action-runner"
+	hostKeyReporterName             = "host-key-reporter"
+	fanConfigurerName               = "fan-configurer"
+	externalControllerUpdaterName   = "external-controller-updater"
+	globalClockUpdaterName          = "global-clock-updater"
+	isResponsibleControllerFlagName = "is-responsible-controller-flag"
+	isControllerFlagName            = "is-controller-flag"
+	logPrunerName                   = "log-pruner"
+	txnPrunerName                   = "transaction-pruner"
```

Comment:
this makes me feel like its reasonable to break this into sections so we don't have to churn the entire table when we add a flag that is slightly longer.